README.md

```diff
@@ -1,13 +1,9 @@
-# jumpstart
+# Jumpstart
 
-Project generator for various languages and frameworks
+Jumpstart is a versatile code template generator that allows you to kickstart your projects with ease.  Simply follow the instructions below to get started. 
```

Comment:
```suggestion
Jumpstart is a versatile code template generator that allows you to kickstart your projects with ease.
```

A one-liner project description is good, devs are aware that the rest of the readme will contain setup/build instrutions.

Reply:
done